lib/genesis/helpers.go

```diff
@@ -82,3 +214,28 @@ func NewGenesisBlockFromTrie(t *trie.Trie) (*types.Header, error) {
 
 	return header, nil
 }
+
+func twoxHash(msg []byte) []byte {
```

Comment:
since this is used here as well as in the runtime `imports_old.go`, could you move this to `common` and update it here and in the runtime imports?

Reply:
Yes.